src/drive/web/modules/filelist/File.jsx

```diff
@@ -262,30 +266,31 @@ const FileAction = forwardRef(function FileAction(
   ref
 ) {
   return (
-    <div
+    <TableCell
       className={cx(
         styles['fil-content-cell'],
         styles['fil-content-file-action'],
         { [styles['fil-content-file-action--disabled']]: isInSyncFromSharing }
       )}
-      ref={ref}
     >
-      <Button
-        theme="action"
-        {...!disabled && { onClick }}
-        extension="narrow"
-        icon={
-          <Icon
-            icon="dots"
-            color={palette.charcoalGrey}
-            width="17"
-            height="17"
-          />
-        }
-        iconOnly
-        label={t('Toolbar.more')}
-      />
-    </div>
+      <span ref={ref}>
```

Comment:
Why creating this span? Is it because TableCell doesn't forwardRef? If yes, we should open an issue in ui.

Reply:
Yes it is. I'll open an issue